synapse/datamodel.py

```diff
@@ -555,6 +559,12 @@ def __init__(self):
             ),
         })
 
+    def setModelVers(self, vers):
+        self.vers = vers
+        callbacks = self.callbacks
+        self.callbacks = []
+        [func(*args, **kwargs) for (func, args, kwargs) in callbacks]
```

Comment:
are these callbacks always going to be sync callbacks?

Reply:
seems likely, but i'm open to adjusting it if there are other cases that would need async callbacks